101-lustre-client-server-create/azuredeploy.json

```diff
@@ -0,0 +1,1358 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "artifactsBaseUrl": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/lustrehpc/azure-quickstart-templates/master/101-lustre-client-server-create",
+      "metadata": {
+        "description": "Base URL of the solution template gallery package",
+        "artifactsBaseUrl": ""
+      }
+    },
+    "location": {
```

Comment:
Since resource group is a logical grouping and the template should still provide option to allow the VM to be created based on location. So we would keep the location parameter with multiple choice.